Source/Forms/Gen5/IDs5.cpp

```diff
@@ -213,7 +224,7 @@ void IDs5::find()
             u64 seed = sha.hashSeed();
 
             generator.setInitialAdvances(flag ? Utilities::initialAdvancesBWID(seed) : Utilities::initialAdvancesBW2ID(seed));
-            auto states = generator.generate(seed);
+            auto states = generator.generate(seed, usePID, useXOR, pid);
```

Comment:
Okay I now remember why it had default parameters. Could you put it back that way it was, my bad.